Add OCR gender filter, scroll support, and cloud vision integration

janssenPeng opened the pull request:
Add OCR gender filter, scroll support, and cloud vision integration

janssenPeng committed:
Add OCR gender filter, scroll support, and cloud vision integration

denisbajgora5 commented:
Please make sure to add multer and @google-cloud/vision to the package.json dependencies. When I ran npm install, those packages weren’t installed, so I had to install them manually. Ideally, all required dependencies should be handled automatically via npm install.

janssenPeng committed:
Add @google-cloud/vision: ^5.1.0 and multer: ^1.4.5-lts.2 into dependencies

denisbajgora5 commented:
Make sure you add "@types/multer": "^1.4.12" in "devDependencies" in package.json. This is so multer works with Typescript.

denisbajgora5 committed:
added multer depency and upgraded project to sdk 53 and it works on expo go now!

Also had to redo the .env system as it wasn't working(had this problem on passenger too)

Only thing left to fix is mobile gender veritfication

denisbajgora5 committed:
Added extra scripts in package.json (forgot to transfer these over)

denisbajgora5 commented:
Last thing to do before squashing and merging into main is to fix gender verification for mobile then we are good to go!

janssenPeng committed:
Complete the verfication for phone end, Add passport OCR with smart gender extraction via MRZ regex

janssenPeng committed:
fix(verification): replace `any` with `unknown as Blob` to satisfy ESLint rule
